src/services/attendees/attendee-schema.ts

```diff
@@ -7,9 +7,11 @@ const AttendeeValidator = z.object({
     name: z.string(),
     email: z.string().email(),
     events: z.array(z.string()),
-    dietary_restrictions: z.string(),
+    dietary_restrictions: z.string().array(),
```

Comment:
follow js naming conventions 

```suggestion
    dietaryRestrictions: z.string().array(),
```